pyteal/ast/itxn_test.py

```diff
@@ -134,4 +135,125 @@ def test_InnerTxnBuilder_SetFields():
                 expr.__teal__(teal4Options)
 
 
+def test_InnerTxnBuilder_method_call():
+    cases = (
```

Comment:
nit: would be nice if it is possible to follow `@pytest.mark.parametrize`? (feel free to ignore if it is hard to do)